config/defines.inc.php

```diff
@@ -91,6 +91,15 @@
     define('_PS_ENV_', _PS_MODE_DEV_ ? 'dev': 'prod');
 }
 
+/* ENV variables config file - depending on app env */
+if (!defined('_PS_ENV_VARS_FILE_')) {
+    if (file_exists(_PS_ROOT_DIR_ . '/env.' . _PS_ENV_)) {
```

Comment:
I think you didn't test it as in previous commits it was `.env` :eyes:

Reply:
you're right. Just tested it and fix it